pom.xml

```diff
@@ -14,7 +14,7 @@
 
     <properties>
         <java.version>1.8</java.version>
-        <kotlin.version>1.2.71</kotlin.version>
+        <kotlin.version>1.3.11</kotlin.version>
```

Comment:
Yes, I know there is the newer version of Kotlin than used in this sample, but this version is recommended by Spring for the 2.1.2.RELEASE of Spring Boot

Reply:
I could not find any place that says that 1.2.71 is "recommended", but it is the version used in the pom file generated by the spring boot configurator for 2.1.2.  Reverted this change.